extensions/blocks/rating-star/style.scss

```diff
@@ -0,0 +1,101 @@
+// Editor and View styles
+.wp-block-jetpack-rating-star,
+.wp-block-jetpack-rating-spiciness,
+.wp-block-jetpack-rating-priciness {
```

Comment:
Would partial match CSS be good here?
```css
[class^=wp-block-jetpack-rating-] { }
```

That would save a lot of CSS being generated. :-)

Reply:
Ended up removing these as per https://github.com/Automattic/jetpack/pull/14091/commits/cc9a951e2b98321aa6307663bab923fd73abba96